subprojects/workers/src/main/java/org/gradle/workers/IsolationMode.java

```diff
@@ -19,22 +19,26 @@
 import org.gradle.api.Incubating;
 
 /**
- * Forking mode for workers.
+ * Isolation mode for workers.
  *
- * @since 3.5
+ * @since 4.0
  */
 @Incubating
-public enum ForkMode {
```

Comment:
should leave `ForkMode` there until _after_ the Android plugin has switched to using `IsolationMode`.